dist/renderer.js

```diff
@@ -0,0 +1,23 @@
+class Renderer {
+
+    renderSearchData(city) {
+        const searchResult = $('#searchResult')
+        searchResult.empty()
+        $('#background').remove()
+        $('body').append($(`
```

Comment:
I would try not to append things to the body. Better to make a container to append to and empty.